eclipse-collections/src/main/java/org/eclipse/collections/impl/utility/internal/InternalArrayIterate.java

```diff
@@ -1214,4 +1226,156 @@ public static <R extends Collection<T>, T> R rejectWithIndex(
         }
         return target;
     }
+
+    /**
+     * Adds all array elements to the target MutableBooleanCollection that return false when evaluating the specified predicate which is
```

Comment:
@Desislav-Petrov The javadoc is incorrect. This looks like the description for reject, not collect.

Reply:
Apologies, my bad - sorted